core/package.json

```diff
@@ -17,7 +17,7 @@
 		"db:prisma": "pnpm prisma",
 		"db:studio": "pnpm prisma studio",
 		"db:generate-history-table": "pnpm exec tsx prisma/scripts/history-tables/generate-history-table.mts",
-		"clear-cache": "rm -rf .next/cache/fetch-cache",
+		"clear-cache": "dotenv -e .env.local -e .env.development -- tsx --eval \"import {getRedisClient} from 'lib/redis'; (async () => {await (await getRedisClient()).flushall(); process.exit(); })();\"",
```

Comment:
this is pretty gross. i'm willing to hear that it should go in a file but i'm not sure where we'd like to put it

Reply:
i think we should just make a `core/scripts` again and put it there, i agree that this is pretty evil